pkg/utils/utils.go

```diff
@@ -61,3 +65,59 @@ func SendToSocket(socket string, command string) error {
 	}
 	return nil
 }
+
+// checkValidity runs a HAProxy configuration validity check on a file
+func checkValidity(configFile string) error {
+	out, err := exec.Command("haproxy", "-c", "-f", configFile).CombinedOutput()
+	if err != nil {
+		glog.Warningf("Error validating config file:\n%v", string(out))
+		return err
+	}
+	return nil
+}
+
+// multibinderERBOnly generates a config file from ERB template by invoking multibinder-haproxy-erb
+func multibinderERBOnly(configFile string) (string, error) {
+	_, err := exec.Command("multibinder-haproxy-erb", "/usr/local/sbin/haproxy", "-f", configFile, "--erb-write-only").CombinedOutput()
```

Comment:
It's safe use `-c` here and remove `--erb-write-only`. This will call HAProxy with a valid .cfg file. Changing this and the output will be useful. Just follow the same approach used in `checkValidity`.